recipes/gwframes/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "GWFrames" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # we have to use git_url here to get the submodules needed for the
+  # build; see <https://github.com/conda-forge/xgboost-feedstock/issues/22>
+  git_url: https://github.com/moble/{{ name }}.git
+  git_rev: 792a341f77fa838bcdfd78940850205bd60c5ba4
+  # Mitigate the above by just cloning the minimum depth
+  git_depth: 1
+
+build:
+  number: 0
+  script: "cd Code && {{ PYTHON }} setup.py install"  # [not win]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pip
+
+  host:
+    - python
+    - setuptools
+    - fftw
+    - gsl
+    - hdf5
+    - numpy
+    - swig =3.0.10
+    - libopenblas  # [linux]
+    - libstdcxx-ng  # [linux]
+    - libgcc-ng  # [linux]
+
+  run:
+    - python
+    - fftw
+    - gsl
+    - hdf5
+    - {{ pin_compatible('numpy') }}
+    - swig =3.0.10
```

Comment:
Do you need SWIG at runtime?
```suggestion
```

Reply:
Ah.  I did, once upon a time.  But it looks like that changed by at least v3.0.  I'll give it a shot.